apps/perf-test/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "perf-test",
+  "name": "@microsoft/perf-test",
```

Comment:
Since the northstar perf-test package was renamed to `perf-test-northstar`, can't this be scoped to `@fluentui/perf-test` instead?

Reply:
hah, i was going to do that AFTER the northstar naming....but obviously forgot